lms/djangoapps/django_comment_client/utils.py

```diff
@@ -522,8 +524,14 @@ def process_view(self, request, view_func, view_args, view_kwargs):
         """
         Injects the view name value into the request context
         """
+
         request.view_name = view_func.__name__
 
+        #  return 404 on the certain URLS listed in site configurations under BLOCKED_URL_NAMES
+        blocked_url_names = configuration_helpers.get_value('BLOCKED_URL_NAMES')
+        if blocked_url_names and resolve(request.path).url_name in blocked_url_names:
+            raise Http404
```

Comment:
Let's make another middleware for the purpose? As the name suggests, this middleware shouldn't do anything except setting the view name.